Tests/Kitura-CredentialsTwitterTests/Kitura-CredentialsTwitterTests.swift

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright Jacob Van Order 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import XCTest
+@testable import Kitura_CredentialsTwitter
+
+var consumerKey = "UdYo5tCHu8HdmY1uIoZxg1kNA"
+var consumerSecret = "MacxkjCjsLgckelqTNQkQBB3WVLKYhb48VZOD7l51rmt1WcFtV"
+var oAuthToken = "z1obMQAAAFAAynC1AAABWaLpD3U"
+var oAuthTokenSecret = "g5qyyHw7txbYaLWZpFgl77wEtjHohgRf"
+var oAuthTokenVerifier = "3lEFtnq9wzKavfSYSnPyntsLOk8FpOkE"
+
+var parametersForRequest: [String: String] = {
```

Comment:
Space after `:`. Although I prefer a space, looks like the Facebook one doesn't have a space in dictionary types.